Bottom: 7a684ccd77496ef7f24b4f63f388bdccb4d863a1
Top:    3a2194cb7df77822b3e14bdf6e2a2aa54cf820ce
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 02:19:36 +0100

action.c: Replace use of obsolete conversion macro TO_PCB with a constant.

This changes the location tolerance in NotifyLine()'s call to
LookupConnection() from what previously evaluated to 100 internal units,
to a constant of one internal unit.

This is consistent with other similar calls to LookupConnection(),
where it appears that a wide search area is not desired, so the smallest
possible value is passed.


---

diff --git a/src/action.c b/src/action.c
index 967a95f..c03042e 100644
--- a/src/action.c
+++ b/src/action.c
@@ -891,7 +891,7 @@ NotifyLine (void)
 	  type = SearchScreen (Crosshair.X, Crosshair.Y,
 			       PIN_TYPE | PAD_TYPE | VIA_TYPE, &ptr1, &ptr2,
 			       &ptr3);
-	  LookupConnection (Crosshair.X, Crosshair.Y, true, TO_PCB (1),
+	  LookupConnection (Crosshair.X, Crosshair.Y, true, 1,
 			    FOUNDFLAG);
 	}
       if (type == PIN_TYPE || type == VIA_TYPE)
